### ISSUE TITLE
Rotten Tomatoes command returns 403 error

### ISSUE THREAD

**Issue by katycat5e** [edwardslabs/CloudBot#213]
.rottentomatoes / .rt command spits out "Error searching: 403" for every input.

\<katycat\> .rt Pixels
\<gonzobot\> Error searching: 403
\<katycat\> .rt Iron Man
\<gonzobot\> Error searching: 403
\<katycat\> .rt Thor: Ragnarok
\<gonzobot\> Error searching: 403
\<katycat\> .rottentomatoes Jigsaw
\<gonzobot\> Error searching: 403

**Comment by Vault108-zz** [comment]
Seems like they recently updated their Policy. 
https://www.fandango.com/policies/privacy-policy
> Last updated: September 27, 2017


They now Suggest using this form to request  Use of the api 
https://www.rottentomatoes.com/help_desk/proposalform 
> Please note that at this time we no longer support unauthorized use of our data (e.g. unofficial projects, non-user facing data integrations). 


This could be whats causing the issue

**Comment by edwardslabs** [comment]
yeah they got rid of their free API. We should remote the plugin.

**Comment by Vault108-zz** [comment]
I know letterboxd is working on an API, would even be useful to switch to that?
Now, while my python is not great I am positive someone in the community could make this work.
Just an idea though
